src/driver/postgres/PostgresConnectionOptions.ts

```diff
@@ -67,4 +67,10 @@ export interface PostgresConnectionOptions extends BaseConnectionOptions, Postgr
      * Automatically install postgres extensions
      */
      readonly installExtensions?: boolean;
+
+    /**
+     * sets the application_name var to help db administrators identify
+     * the service using this connection. Defaults to 'typeorm app'
```

Comment:
```suggestion
     * the service using this connection. Defaults to 'undefined'
```

Reply:
yeah saw that a bit too late. i've added it while you where reviewing i think, sorry